src/main/webapp/main/LandingPage.js

```diff
@@ -119,15 +124,19 @@ const LandingPage = hh(class LandingPage extends Component{
         projectList: projectList,
         taskList: taskList,
         logged: user.data.session
-      })
+      });
+      Storage.setUserIsLogged(user.data.session);
     }
   };
 
   render() {
+    if (Storage.userIsLogged() && !this.state.logged) {
+      this.init();
+    }
     return (
       div({}, [
         About(),
-        div({className: "row", isRendered: this.state.logged && component.isBroad}, [
+        div({className: "row", isRendered: Storage.userIsLogged() &&  !nonBroadUser()}, [
```

Comment:
Instead of the negation `!nonBroadUser()`, can we use `isBroad` here? It would also make sense that `isBroad` would imply that you are also logged in, so these should, in theory, be combined into a single method call.